server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ShowPlan.java

```diff
@@ -39,6 +39,11 @@ public ShowPlan(ShowContentType showContentType){
     return null;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    return null;
```

Comment:
And here.

Reply:
I'll change it. Thanks for your suggestion.